src/dtype.rs

```diff
@@ -94,12 +90,207 @@ impl PyArrayDescr {
         }
     }
 
-    /// Retrieves the
-    /// [enumerated type](https://numpy.org/doc/stable/reference/c-api/dtype.html#enumerated-types)
-    /// for this type descriptor.
-    pub fn get_typenum(&self) -> c_int {
```

Comment:
Similar q about deprecation here.

Reply:
This one ^ I don't think we need to deprecate because it doesn't exist in the latest stable (0.15.1).